scripts/ayp.coffee

```diff
@@ -238,30 +238,30 @@ module.exports = (robot) ->
           ]
           msg.reply "#{prefix} #{url} is now -> #{strip.info.image_url}"
 
-          # Tweet a link to the comic, with some sort of text to go with it.
-          prefix = msg.random [
-            "Welp",
-            "Here you go",
-            "You made this? I made this.",
-            "The one that doesn't make sense",
-            "In which nothing happens",
-            "Laughter, sorrow, nonsense",
-            "#arrakis",
-            "#dickbutt",
-            "butts",
-            "Have a comic",
-            "#pants",
-          ]
-          Twitter.mediaTweet strip.info.image_jpeg, "#{prefix} - #{strip.info.url}", (err, tweet, url) =>
-            sorry = msg.random ["sorry", "it's your fault"]
-            return msg.reply "Well, I fucked tweeting that RIGHT up, #{sorry}: #{err}" if err
-            prefix = msg.random [
-              "SHARE FAVE RT",
-              "Connect with #brands",
-              "Promoted by #a",
-              "I put it on twitter, shitbirds",
-            ]
-            msg.send "#{prefix} #{url}"
+#           # Tweet a link to the comic, with some sort of text to go with it.
+#           prefix = msg.random [
+#             "Welp",
+#             "Here you go",
+#             "You made this? I made this.",
+#             "The one that doesn't make sense",
+#             "In which nothing happens",
+#             "Laughter, sorrow, nonsense",
+#             "#arrakis",
+#             "#dickbutt",
+#             "butts",
+#             "Have a comic",
+#             "#pants",
+#           ]
+#           Twitter.mediaTweet strip.info.image_jpeg, "#{prefix} - #{strip.info.url}", (err, tweet, url) =>
+#             sorry = msg.random ["sorry", "it's your fault"]
+#             return msg.reply "Well, I fucked tweeting that RIGHT up, #{sorry}: #{err}" if err
```

Comment:
Maybe instead of a massive comment block, we can do `return msg.reply "Twitter is still shadowbanned :("` right above this line (or even just `return` to silently eat the error) to avoid throwing a large block back and forth? Then if and when we can get working keys it would just start working again, but less noisy?

Reply:
@sshirokov I went for the quiet return, I'd rather have it say nothing than continue to repeat the same line over and over. Unless we're clever enough to come up with an array of ways to say it failed.